<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="scanerPanel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="scannerInitBT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Init</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scanBT">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QLabel" name="scanImg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>320</width>
       <height>420</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="scannerOutput">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scanner</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>66</y>
       <width>721</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="skeletonImg">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>320</width>
       <height>420</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Response From Server</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>20</width>
       <height>491</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="ipTF">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ipLB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portTF">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBT">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLB">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBT">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectBT">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QFrame" name="params">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>521</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="gaborFilterParamsPanel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="blockSizeLB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Block size</string>
      </property>
     </widget>
     <widget class="QLabel" name="lambdaLB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lambda</string>
      </property>
     </widget>
     <widget class="QLabel" name="sigmaLB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sigma</string>
      </property>
     </widget>
     <widget class="QLabel" name="gammaLB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gamma</string>
      </property>
     </widget>
     <widget class="QLabel" name="psiLB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Psi</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="lambda">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <double>9.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sigma">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="gamma">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="psi">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="blockSize">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>46</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>13</number>
      </property>
     </widget>
     <widget class="QLabel" name="gaborFilterLB">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gabor filter params</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="gaborFilterParamsPanel_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>261</width>
       <height>411</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="blockSizeLB_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Block size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="gaussBasicBlockSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>46</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="gaussFilterLB_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gauss filter params</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gaussBasicFilterLB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Basic</string>
      </property>
     </widget>
     <widget class="QLabel" name="sigmaLB_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sigma</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="gaussBasicSigma">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="gaussAdvancedBlockSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>46</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>121</number>
      </property>
     </widget>
     <widget class="QLabel" name="blockSizeLB_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Block size</string>
      </property>
     </widget>
     <widget class="QLabel" name="sigmaLB_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sigma</string>
      </property>
     </widget>
     <widget class="QLabel" name="gaussBasicFilterLB_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Advanced</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="gaussAdvancedSigma">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="holeSizeLB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hole size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="holeSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>46</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="returnSkeleton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>return skeleton fingerprint</string>
     </property>
    </widget>
    <widget class="QPushButton" name="defaultParamsBT">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>457</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </widget>
   <zorder>params</zorder>
   <zorder>statusLB</zorder>
   <zorder>scanerPanel</zorder>
   <zorder>ipTF</zorder>
   <zorder>ipLB</zorder>
   <zorder>portTF</zorder>
   <zorder>connectBT</zorder>
   <zorder>portLB</zorder>
   <zorder>sendBT</zorder>
   <zorder>disconnectBT</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1322</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
